channeldb/graph.go

```diff
@@ -4666,7 +4746,7 @@ func deserializeChanEdgePolicy(r io.Reader,
 	node, err := fetchLightningNode(nodes, pub)
 	if err != nil {
 		return nil, fmt.Errorf("unable to fetch node: %x, %v",
-			pub[:], err)
+			pub, err)
```

Comment:
nit: fix in previous commit and re-flow to fit on one line.

Reply:
fixed